ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -140,6 +142,29 @@ export class AptosAccount {
     return this.signBuffer(toSign);
   }
 
+  /**
+   * Takes source address and seeds and returns the resource account address
+   * @param sourceAddress Address used to derive the resource account
+   * @param seeds The seeds need to be in hexadecimal format
+   * @returns The resource account address
+   */
+
+  getResourceAccountAddress(sourceAddress: MaybeHexString, seeds: string): HexString {
```

Comment:
Should be a static method?

Reply:
Also, it is better to make seeds Uint8Array

Reply:
Yes it should be a static method.

Reply:
ok ill make it

Reply:
> Also, it is better to make seeds Uint8Array

Ill make seeds Uint8Array